app/src/main/java/com/work/restaurant/view/search/lookfor/adapter/LookForAdapter.kt

```diff
@@ -1,19 +1,17 @@
-package com.work.restaurant.view.search.adapter
+package com.work.restaurant.view.search.lookfor.adapter
 
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.ImageView
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 import com.work.restaurant.R
-import com.work.restaurant.network.model.FitnessCenterItemResponse
-import com.work.restaurant.util.GlideApp
+import com.work.restaurant.data.model.KakaoSearchModel
 import com.work.restaurant.view.adapter.AdapterDataListener
 
 class LookForAdapter : RecyclerView.Adapter<LookForAdapter.ViewHolder>() {
 
-    private val searchLookList = ArrayList<FitnessCenterItemResponse>()
+    private val searchLookList = ArrayList<KakaoSearchModel>()
```

Comment:
```suggestion
    private val searchLookList = listOf<KakaoSearchModel>()
```
List가 아닌 ArrayList를 사용해야 할 이유가 있을까요?

Reply:
fb49a24 수정하였습니다.